keras/layers/wrappers.py

```diff
@@ -539,6 +539,8 @@ def call(self,
             output = y * y_rev
         elif self.merge_mode is None:
             output = [y, y_rev]
+        else:
+            raise ValueError('Unrecognized merge_mode ' + self.merge_mode)
```

Comment:
Please use string formatting:

'Unrecognized value for argument `merge_mode`: %s' % (self.merge_mode,)